addon/tailwind/components/bourbon-text-field.css

```diff
@@ -52,6 +52,7 @@ input[type="search"].BourbonTextField {
   white-space: nowrap;
   width: auto;
   z-index: 1;
+  visibility: hidden;
```

Comment:
How well does this work with the `transition` property? I don't remember how transitionable the `visibility` property is.

Reply:
the transition property is still working fine with the opacity which is the one that provides the nice easing in.  :)

Reply:
looks like browsers are smart enough now that the transitions work both ways 👍